versions/development/SPEC.md

```diff
@@ -1986,54 +1961,54 @@ struct Person {
 then usage of the struct in a workflow would look like the following:
 
 ```wdl
-
-task task_a {
-    Person a
-    command {
-        echo "hello my name is ${a.name} and I am ${a.age} years old"
+task myTask {
+    input {
+      Person a
+    }
+    command <<<
+        echo "hello my name is ~{a.name} and I am ~{a.age} years old"
+    >>>
+    output {
+      String name = a.name + "Potter"
+      Int age = a.age * 2
     }
 }
 
 workflow myWorkflow {
-    Person a
-    call task_a {
+    Person harry = object {name: "Harry", age: 11}
+    call myTask {
         input:
-            a = a
+            a = harry
     }
+    Person harry_p = myTask
```

Comment:
Typo: `Person harry_p = harry`?

Reply:
With this one I was trying to illustrate how tasks return is compatible with Struct.